lib/discourse_api/api/groups.rb

```diff
@@ -115,9 +115,8 @@ def group_members(group_name, params = {})
         response.body['members']
       end
 
-      def group_set_user_notification_level(group, user_id, notification_level)
-        response = post("/groups/#{group}/notifications?user_id=#{user_id}&notification_level=#{notification_level}")
-        response
+      def group_set_user_notify_level(group, user_id, notification_level)
```

Comment:
Could you explain the reasoning behind changing the method name from "notification" to "notify"?

Unless there is a good reason we should probably keep it "notification" to not break anything for existing users?

Reply:
@oblakeerickson what direction would you like to go here so we can close out this PR?

Reply:
Oh sorry for letting this go so long, let's not change the method name.